recipe/meta.yaml

```
@@ -12,9 +12,11 @@ source:
     # Taken from https://github.com/microsoft/LightGBM/blob/v4.5.0/build-python.sh#L308-L316
     # except that we don't copy lib_lightgbm.so to SP_DIR
     - use_precompiled.diff
+    # Compatibility fix for Boost 1.86
+    - boost-sha1.diff
 
 build:
-  number: 4
+  number: 6
```

```suggestion
  number: 5
```

There aren't any published `*_5` packages, so assuming this was a mistake.